packages/lisk-transaction-pool/src/job.ts

```diff
@@ -0,0 +1,50 @@
+export class Job<T> {
+	private _id: NodeJS.Timer | undefined;
+	private readonly _interval: number;
+	private readonly _job: () => Promise<T>;
+	private _running = false;
+
+	public constructor(
+		job: () => Promise<T>,
+		interval: number,
+	) {
+		this._interval = interval;
+		this._job = job;
+	}
+
+	public async start(): Promise<void> {
+		if (!this._running) {
+			this._running = true;
+
```

Comment:
remove empty line

Reply:
We have a lint rule that there should be an empty line before a return statement.

Reply:
Kk.